<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Human DNA polymorphisms - Module | VOER</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <!-- Bootstrap -->
    <link href="bootstrap/css/bootstrap.css" rel="stylesheet" media="screen">
    <link href="bootstrap/css/bootstrap-blue.css" rel="stylesheet" media="screen">
    <link href="bootstrap/css/bootstrap-responsive.css" rel="stylesheet">
    <link href="css/jui/blitzer/jquery-ui-1.10.1.custom.css" rel="stylesheet">
    <link href="css/style.css" rel="stylesheet" media="screen">
    <link href="css/abc.css" rel="stylesheet" media="screen">
	<link rel="stylesheet" href="js/jRating.jquery.css" type="text/css" />
  </head>
  <body data-offset="80" data-target=".subnav" data-spy="scroll" style="height:1000px">
    <div class="navbar navbar-fixed-top">
      <div class="navbar-inner">
        <div class="container">
          <a id="logo" href="index.html"><img src="images/voer_logo.png" /></a>
          <ul class="nav">
            <li><a href="#">Learn</a></li>
            <li><a href="#">Contribute</a></li>
            <li><a href="#">About us</a></li>
            <li><a href="#">Support</a></li>
          </ul>
          <ul class="nav pull-right">
            <li><a class="dashboard" href="#">Dashboard</a></li>
            <li><a href="#">Sign out</a></li>
          </ul>

        </div>
      </div>
    </div>

    <div class="container">
      <div class="row">
        <div class="span3">
          <div class="voer-sidenav1" data-spy="affix" data-offset-top="5">
            <div class="well">
              <ul class="nav nav-list">
                <li class="nav-header">Related modules</li>
                <li><a href="#">About the controversal ethical issues on applications of genetics</a></li>
                <li><a href="#">Genetics in clasical understanding</a></li>
                <li><a href="#">Applications and design of integrated circuits</a></li>
                <li><a href="#">Basic Principles of Genetics</a></li>
				<li><a href="#">Genetic complex traits</a></li>
                <li><a href="#">Mendelian discovery of genes</a></li>
                <li><a href="#">Gene linkage</a></li>
                <li><a href="#">Probability and pedigrees</a></li>
              </ul>
            </div>
          </div>
        </div>
        <div class="span9">
          <ul class="breadcrumb">
            <li><a href="index.html">Home</a> <span class="divider">/</span></li>
            <li><a href="#">Module</a> <span class="divider">/</span></li>
            <li><a href="#">Biology & Life Sciences</a> <span class="divider">/</span></li>
            <li class="active">Human DNA polymorphisms</li>
          </ul>
		  
          <div class="page-header"><h1>Human DNA polymorphisms</h1></div>
		  <div class="basic">
			
		  </div>
          <div class="btn-toolbar">
          <div class="btn-group">
          <a href="#" class="btn btn-inverse disabled"><i class="icon-white icon-thumbs-up"></i></a>
          <a href="#" class="btn btn-inverse disabled"><i class="icon-white icon-heart"></i></a>
          <a href="#" class="btn btn-inverse disabled"><i class="icon-white icon-download-alt"></i></a>
          <a href="#" class="btn btn-inverse disabled"><i class="icon-white icon-share-alt"></i></a>
          </div>
          </div>
		<div class="image" style="float:right; margin: 10px;">
			<img src="./images/graphics5.png" width="188" height="234" data-mime-type="image/png" alt="DNA structure" />
			<div class="caption" style="width:190px; margin-top:5px;font-style:italic;font-size:10pt;text-align:justify;">A single nucleotide polymorphism (SNP, pronounced snip) is a DNA sequence variation</div>
		</div>
		<p id="id1727888" style="text-align: justify;">		
		One of the most important tools underlying the revolution in medical genetics is the ability to visualize sequence differences directly in DNA. When studied in the context of a population, these differences in DNA sequences are called polymorphisms; they may occur in coding regions (exons) or noncoding regions of genes. The ability to visualize thousands of DNA polymorphisms has made possible family studies for tracking genes of medical importance. This technique has located and identified genes for many disorders with a clear pattern of mendelian inheritance, such as cystic fibrosis, the inherited muscular dystrophies, and neurodegenerative disorders such as Huntington's disease. Methods that exploit genetic polymorphism will also be essential for finding genes that predispose people to more common conditions in which inheritance patterns are complex, such as diabetes, atherosclerosis, and hypertension.</p>
		<p id="id1728033" style="text-align: justify;">DNA polymorphisms are also playing a crucial part in unraveling the genetic basis of tumor formation and progression in cancer. They provide markers for the loss of specific chromosomal segments during the evolution of a tumor. DNA polymorphisms have already been crucial in the identification of genes important for susceptibility to common forms of cancer, such as colon cancer, as well as susceptibility to less common childhood tumors, such as retinoblastoma and Wilms' tumor.</p>
		<p id="id1728112" style="text-align: justify;">The most useful DNA sequence polymorphisms have many alternative forms. The value of highly variable DNA sequences as genetic markers rests on straightforward principles. Every person carries two copies of each chromosome except the sex chromosomes. If a DNA polymorphism is to be useful in analyzing the transmission of the two chromosomes in a family or the loss of one of the chromosomes during tumorigenesis, then the DNA copies at the polymorphic site of the person under study must be different in the two chromosomes (<a href="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1" data-url="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1">Figure 1A</a>),&nbsp;<a href="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1" data-url="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1">Figure 1B</a>),&nbsp;<a href="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1" data-url="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1">Figure 1C</a>), and&nbsp;<a href="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1" data-url="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1">Figure 1D</a>). The likelihood that a given person will have different DNA sequences at the polymorphic site directly determines the usefulness of that site in genetic studies. Chromosomal sites at which the DNA sequences can have many alternative forms are thus ideal sites for genetic markers. At these sites, a person is most likely to carry two alternative DNA sequences, accurately marking the two alternative chromosomes.</p>
		<p id="id1728326" style="text-align: justify;">In the human genome, the sites that have the properties most favorable to such extensive variation include a repetition of the same short DNA sequence a variable number of times. Such sequences are called tandem-repeat sequences. A DNA sequence with such variation may be as short as two base pairs or as long as several hundred base pairs. Highly variable sequences of this type are well distributed throughout the length of every human chromosome. When tandemly repeated sequences are replicated during cell division, the number of repeats can change. The frequency of this kind of replication error is high enough to make alternative lengths at the polymorphic site common, but the rate of change in the length of the site is low enough that the size of the DNA at the polymorphic site serves as a stable trait in family studies (<a href="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1" data-url="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1">Figure 1A</a>).</p>
		<p id="id1728480" style="text-align: justify;">Two techniques, Southern blotting and the polymerase chain reaction (PCR), can measure the length of the DNA sequence at the polymorphic site (<a href="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1" data-url="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1">Figure 1B</a>). The one to choose depends on the length of the tandemly repeated sequence. A repeated sequence 20 to 40 base pairs in length leads to variation in DNA lengths of hundreds or even thousands of base pairs at the polymorphic site. Southern blotting is best for visualizing this degree of variation in length. Very short tandemly repeated sequences, only two, three, or four base pairs long, can also vary highly. For these, the PCR is preferred. Whichever technique is used, its goal is to assess accurately the length of the DNA segment between two fixed points on each chromosome. These two points include some DNA adjacent to the repeated sequence as well as the repeated sequence itself. In the case of Southern blotting, the position of the fixed points is defined by the location of restriction-enzyme digestion sites in the DNA flanking the repeated sequence. In the case of PCR, the positions in the flanking DNA of sequences homologous to the oligonucleotide PCR primers define the fixed points.</p>
		<p id="id1728685" style="text-align: justify;">In Southern blotting, the DNA isolated from each patient or tumor to be typed is digested with a restriction enzyme, separated on the basis of size by agarose-gel electrophoresis, and transferred to a nylon membrane. A DNA probe can reveal directly on the nylon membrane the size of DNA fragments carrying the repeated sequence. This probe corresponds to a sequence in the DNA flanking the repeated sequence. In general, DNA from one person shows two such DNA fragments or bands (<a href="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1" data-url="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1">Figure 1C</a>). For each chromosomal site, one of the two bands will be passed to the next generation, and the other will not, thus indicating the outcome in genetic transmission that occurred at this particular chromosomal site.</p>
		<p id="id1728808" style="text-align: justify;">With the PCR method, the unique sites of primer binding adjacent to the repeated sequence allow specific amplification of the region that includes the repeat. The size of the amplified DNA molecules representing the polymorphic site can now be determined with the same technique that determines the DNA sequence. Precise determination of the length of the amplified DNA molecules usually shows two alternative copies of the DNA fragment, one for each of the chromosomes on which that sequence resides. The application of the two techniques has varied somewhat in human genetic studies; each has advantages and limitations. Sites of short sequence-length variation have been found to be widely distributed along the chromosomes, making them the most widely used sites in genetic-linkage studies designed to track medically important genes in families.</p>
		<p id="id1728935" style="text-align: justify;">Studies of tumors must compare the DNA of normal cells with that of cancer cells. The normal cells usually have two bands, whereas the tumor cells often have only one. This finding is diagnostic of the loss of one copy of a chromosomal region during tumorigenesis. The problem of contamination of a tumor by normal cells presents important issues for studies of this type. Because the PCR involves an amplification process, the amount of material in the starting sample and the amount present in the final amplification product are not necessarily linearly related. Making a judgment about the loss of chromosomal material in a tumor sample contaminated with a substantial number of cells from surrounding normal tissue can be quite challenging. Unlike the results of the PCR, the signal generated by the Southern blotting procedure is directly proportional to the relative amount of each allele present in a tumor sample. Southern blotting has thus been used with particular effect in studies of the loss of chromosomal material by tumor cells (<a href="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1" data-url="http://content.nejm.org/cgi/content/full/332/5/318#F1#F1">Figure 1D</a>).</p>
		<p id="id1729114" style="text-align: justify;">Genetic mapping can determine the relative positions of highly variable DNA sites on each chromosome. Well-characterized polymorphic DNA sites now number in the thousands. The availability of this large number of closely spaced genetic markers has revolutionized human genetics, because it allows the application of genetic-mapping strategies with great precision. For many medically important genes, particularly those that contribute to a predisposition to common medical conditions, the primary limitation to their identification was until recently the availability of a sufficient number of highly informative genetic markers. The techniques described here have removed this limitation. As a result, many important developments in all aspects of medicine are likely to follow.</p>
		<p id="id1729254" style="text-align: justify;">One more important class of DNA polymorphism is single-nucleotide polymorphism</p>
		<p style="text-align: justify;">A single-nucleotide polymorphism (SNP, pronounced snip) is a&nbsp;<a href="http://en.wikipedia.org/wiki/DNA_sequence" data-url="http://en.wikipedia.org/wiki/DNA_sequence">DNA sequence</a>&nbsp;variation occurring when a single&nbsp;<a href="http://en.wikipedia.org/wiki/Nucleotide" data-url="http://en.wikipedia.org/wiki/Nucleotide">nucleotide</a>—&nbsp;<a href="http://en.wikipedia.org/wiki/Adenine" data-url="http://en.wikipedia.org/wiki/Adenine">A</a>,&nbsp;<a href="http://en.wikipedia.org/wiki/Thymine" data-url="http://en.wikipedia.org/wiki/Thymine">T</a>,&nbsp;<a href="http://en.wikipedia.org/wiki/Cytosine" data-url="http://en.wikipedia.org/wiki/Cytosine">C</a>, or&nbsp;<a href="http://en.wikipedia.org/wiki/Guanine" data-url="http://en.wikipedia.org/wiki/Guanine">G</a>&nbsp;— in the&nbsp;<a href="http://en.wikipedia.org/wiki/Genome" data-url="http://en.wikipedia.org/wiki/Genome">genome</a>&nbsp;(or other shared sequence) differs between members of a species (or between paired chromosomes in an individual). For example, two sequenced DNA fragments from different individuals, AAGCCTA to AAGCTTA, contain a difference in a single nucleotide. In this case we say that there are two&nbsp;<a href="http://en.wikipedia.org/wiki/Allele" data-url="http://en.wikipedia.org/wiki/Allele">alleles</a>&nbsp;: C and T. Almost all common SNPs have only two alleles.</p>
		<p style="text-align: justify;">&nbsp;</p>
		<p id="id1729460" style="text-align: justify;">DNA molecule 1 differs from DNA molecule 2 at a single base-pair location (a C/T polymorphism).Within a population, SNPs can be assigned a&nbsp;<a href="http://en.wikipedia.org/wiki/Minor_allele_frequency" data-url="http://en.wikipedia.org/wiki/Minor_allele_frequency">minor allele frequency</a>&nbsp;— the lowest allele frequency at a&nbsp;<a href="http://en.wikipedia.org/wiki/Locus_(genetics)" data-url="http://en.wikipedia.org/wiki/Locus_(genetics)">locus</a>&nbsp;that is observed in a particular population. This is simply the lesser of the two allele frequencies for single-nucleotide polymorphisms. There are variations between human populations, so a SNP allele that is common in one geographical or ethnic group may be much rarer in another.</p>
		<p id="id1729529" style="text-align: justify;">In the past, SNPs with a minor allele frequency of greater than or equal to 1% (or 0.5%, etc.) were given the title "SNP".<a href="http://en.wikipedia.org/wiki/Single_nucleotide_polymorphism#cite_note-0#cite_note-0" data-url="http://en.wikipedia.org/wiki/Single_nucleotide_polymorphism#cite_note-0#cite_note-0">[1]</a>&nbsp;Some used "<a href="http://en.wikipedia.org/wiki/Mutation" data-url="http://en.wikipedia.org/wiki/Mutation">mutation</a>" to refer to variations with low allele frequency. With the advent of modern&nbsp;<a href="http://en.wikipedia.org/wiki/Bioinformatics" data-url="http://en.wikipedia.org/wiki/Bioinformatics">bioinformatics</a>and a better understanding of evolution, this definition is no longer necessary, e.g., a database such as&nbsp;<a href="http://en.wikipedia.org/wiki/DbSNP" data-url="http://en.wikipedia.org/wiki/DbSNP">dbSNP</a>&nbsp;includes "SNPs" that have lower allele frequency than one percent.<a href="http://en.wikipedia.org/wiki/Single_nucleotide_polymorphism#cite_note-1#cite_note-1" data-url="http://en.wikipedia.org/wiki/Single_nucleotide_polymorphism#cite_note-1#cite_note-1">[2]</a></p>
		<p id="id1729621" style="text-align: justify;">Single-nucleotide&nbsp;<a href="http://en.wikipedia.org/wiki/Polymorphism_(biology)" data-url="http://en.wikipedia.org/wiki/Polymorphism_(biology)">polymorphisms</a>&nbsp;may fall within coding sequences of&nbsp;<a href="http://en.wikipedia.org/wiki/Gene" data-url="http://en.wikipedia.org/wiki/Gene">genes</a>,&nbsp;<a href="http://en.wikipedia.org/wiki/Intron" data-url="http://en.wikipedia.org/wiki/Intron">non-coding regions of genes</a>, or in the<a href="http://en.wikipedia.org/wiki/Intergenic_region" data-url="http://en.wikipedia.org/wiki/Intergenic_region">intergenic regions</a>&nbsp;between genes. SNPs within a coding sequence will not necessarily change the&nbsp;<a href="http://en.wikipedia.org/wiki/Amino_acid" data-url="http://en.wikipedia.org/wiki/Amino_acid">amino acid</a>&nbsp;sequence of the&nbsp;<a href="http://en.wikipedia.org/wiki/Protein" data-url="http://en.wikipedia.org/wiki/Protein">protein</a>&nbsp;that is produced, due to&nbsp;<a href="http://en.wikipedia.org/wiki/Genetic_code#Degeneracy_of_the_genetic_code" data-url="http://en.wikipedia.org/wiki/Genetic_code#Degeneracy_of_the_genetic_code">degeneracy of the genetic code</a>. A SNP in which both forms lead to the same polypeptide sequence is termed&nbsp;<a href="http://en.wikipedia.org/wiki/Synonymous_mutation" data-url="http://en.wikipedia.org/wiki/Synonymous_mutation">synonymous</a>&nbsp;(sometimes called a&nbsp;<a href="http://en.wikipedia.org/wiki/Silent_mutation" data-url="http://en.wikipedia.org/wiki/Silent_mutation">silent mutation</a>) — if a different polypeptide sequence is produced they are&nbsp;<a href="http://en.wikipedia.org/w/index.php?title=Nonsynonymous_mutation&amp;action=edit&amp;redlink=1" data-url="http://en.wikipedia.org/w/index.php?title=Nonsynonymous_mutation&amp;action=edit&amp;redlink=1">nonsynonymous</a>. A nonsynonymous change may either be&nbsp;<a href="http://en.wikipedia.org/wiki/Missense_mutation" data-url="http://en.wikipedia.org/wiki/Missense_mutation">missense</a>&nbsp;or&nbsp;<a href="http://en.wikipedia.org/wiki/Nonsense_mutation" data-url="http://en.wikipedia.org/wiki/Nonsense_mutation">nonsense</a>, where a missense change results in a different amino acid, while a nonsense change results in a premature&nbsp;<a href="http://en.wikipedia.org/wiki/Stop_codon" data-url="http://en.wikipedia.org/wiki/Stop_codon">stop codon</a>. SNPs that are not in protein-coding regions may still have consequences for&nbsp;<a href="http://en.wikipedia.org/wiki/Gene_splicing" data-url="http://en.wikipedia.org/wiki/Gene_splicing">gene splicing</a>,&nbsp;<a href="http://en.wikipedia.org/wiki/Transcription_factor" data-url="http://en.wikipedia.org/wiki/Transcription_factor">transcription factor</a>&nbsp;binding, or the sequence of&nbsp;<a href="http://en.wikipedia.org/wiki/Non-coding_RNA" data-url="http://en.wikipedia.org/wiki/Non-coding_RNA">non-coding RNA</a>.</p>
		<p id="id1729886" style="text-align: justify;">Variations in the DNA sequences of humans can affect how humans develop&nbsp;<a href="http://en.wikipedia.org/wiki/Disease" data-url="http://en.wikipedia.org/wiki/Disease">diseases</a>&nbsp;and respond to&nbsp;<a href="http://en.wikipedia.org/wiki/Pathogen" data-url="http://en.wikipedia.org/wiki/Pathogen">pathogens</a>,<a href="http://en.wikipedia.org/wiki/Chemical" data-url="http://en.wikipedia.org/wiki/Chemical">chemicals</a>,&nbsp;<a href="http://en.wikipedia.org/wiki/Medication" data-url="http://en.wikipedia.org/wiki/Medication">drugs</a>,&nbsp;<a href="http://en.wikipedia.org/wiki/Vaccine" data-url="http://en.wikipedia.org/wiki/Vaccine">vaccines</a>, and other agents. SNPs are also thought to be key enablers in realizing the concept of<a href="http://en.wikipedia.org/wiki/Personalized_medicine" data-url="http://en.wikipedia.org/wiki/Personalized_medicine">personalized medicine</a>.<a href="http://en.wikipedia.org/wiki/Single_nucleotide_polymorphism#cite_note-2#cite_note-2" data-url="http://en.wikipedia.org/wiki/Single_nucleotide_polymorphism#cite_note-2#cite_note-2">[3]</a>&nbsp;However, their greatest importance in biomedical research is for comparing regions of the genome between&nbsp;<a href="http://en.wikipedia.org/wiki/Cohorts" data-url="http://en.wikipedia.org/wiki/Cohorts">cohorts</a>&nbsp;(such as with matched cohorts with and without a disease).</p>
		<p id="id1730026" style="text-align: justify;">The study of single-nucleotide polymorphisms is also important in crop and livestock breeding programs (see&nbsp;<a href="http://en.wikipedia.org/wiki/Genotyping" data-url="http://en.wikipedia.org/wiki/Genotyping">genotyping</a>). See&nbsp;<a href="http://en.wikipedia.org/wiki/SNP_genotyping" data-url="http://en.wikipedia.org/wiki/SNP_genotyping">SNP genotyping</a>&nbsp;for details on the various methods used to identify SNPs.</p>
		<p id="id1730067" style="text-align: justify;">Microsatellites&nbsp;Longer DNA sequence repeats are Microsatellites, or Simple Sequence Repeats (SSRs called also STRs), which are&nbsp;<a href="http://en.wikipedia.org/wiki/Polymorphism_(biology)" data-url="http://en.wikipedia.org/wiki/Polymorphism_(biology)">polymorphic</a>&nbsp;loci present in&nbsp;<a href="http://en.wikipedia.org/wiki/Nuclear_DNA" data-url="http://en.wikipedia.org/wiki/Nuclear_DNA">nuclear</a>&nbsp;and&nbsp;<a href="http://en.wikipedia.org/wiki/Organellar_DNA" data-url="http://en.wikipedia.org/wiki/Organellar_DNA">organellar</a><a href="http://en.wikipedia.org/wiki/DNA" data-url="http://en.wikipedia.org/wiki/DNA">DNA</a>&nbsp;that consist of repeating units of 1- 6&nbsp;<a href="http://en.wikipedia.org/wiki/Base_pair" data-url="http://en.wikipedia.org/wiki/Base_pair">base pairs</a>&nbsp;in length.&nbsp;<a href="http://en.wikipedia.org/wiki/Microsatellite#cite_note-Turnpenny-0#cite_note-Turnpenny-0" data-url="http://en.wikipedia.org/wiki/Microsatellite#cite_note-Turnpenny-0#cite_note-Turnpenny-0">[1]</a>&nbsp;They are typically neutral,&nbsp;<a href="http://en.wikipedia.org/wiki/Co-dominant" data-url="http://en.wikipedia.org/wiki/Co-dominant">co-dominant</a>&nbsp;and are used as&nbsp;<a href="http://en.wikipedia.org/wiki/Molecular_marker" data-url="http://en.wikipedia.org/wiki/Molecular_marker">molecular markers</a>&nbsp;which have wide-ranging applications in the field of&nbsp;<a href="http://en.wikipedia.org/wiki/Genetics" data-url="http://en.wikipedia.org/wiki/Genetics">genetics</a>, including&nbsp;<a href="http://en.wikipedia.org/wiki/Kinship" data-url="http://en.wikipedia.org/wiki/Kinship">kinship</a>&nbsp;and&nbsp;<a href="http://en.wikipedia.org/wiki/Population" data-url="http://en.wikipedia.org/wiki/Population">population</a>&nbsp;studies. Microsatellites can also be used to study gene dosage (looking for&nbsp;<a href="http://en.wikipedia.org/wiki/Gene_duplication" data-url="http://en.wikipedia.org/wiki/Gene_duplication">duplications</a>&nbsp;or&nbsp;<a href="http://en.wikipedia.org/wiki/Genetic_deletion" data-url="http://en.wikipedia.org/wiki/Genetic_deletion">deletions</a>&nbsp;of a particular genetic region).</p>
		<p id="id41035531" style="text-align: justify;">One rare example of a microsatellite is a (CA)n repeat, where n is variable between&nbsp;<a href="http://en.wikipedia.org/wiki/Alleles" data-url="http://en.wikipedia.org/wiki/Alleles">alleles</a>. These markers often present high levels of inter- and intra-specific polymorphism, particularly when tandem repeats number one hundred or greater.<a href="http://en.wikipedia.org/wiki/Microsatellite#cite_note-goku-1#cite_note-goku-1" data-url="http://en.wikipedia.org/wiki/Microsatellite#cite_note-goku-1#cite_note-goku-1">[2]</a>&nbsp;The repeated sequence is often simple, consisting of two, three or four&nbsp;<a href="http://en.wikipedia.org/wiki/Nucleotides" data-url="http://en.wikipedia.org/wiki/Nucleotides">nucleotides</a>&nbsp;(di-, tri-, and tetranucleotide repeats respectively), and can be repeated 10 to 100 times. CA nucleotide repeats are very frequent in<a href="http://en.wikipedia.org/wiki/Human" data-url="http://en.wikipedia.org/wiki/Human">human</a>&nbsp;and other&nbsp;<a href="http://en.wikipedia.org/wiki/Genome" data-url="http://en.wikipedia.org/wiki/Genome">genomes</a>, and are present in every few thousand base pairs. As there are often many alleles present at a microsatellite locus,&nbsp;<a href="http://en.wikipedia.org/wiki/Genotype" data-url="http://en.wikipedia.org/wiki/Genotype">genotypes</a>&nbsp;within&nbsp;<a href="http://en.wikipedia.org/wiki/Pedigree_chart" data-url="http://en.wikipedia.org/wiki/Pedigree_chart">pedigrees</a>&nbsp;are often fully informative, in that the&nbsp;<a href="http://en.wikipedia.org/wiki/Progenitor" data-url="http://en.wikipedia.org/wiki/Progenitor">progenitor</a>&nbsp;of a particular allele can often be identified. In this way, microsatellites are ideal for determining paternity, population genetic studies and<a href="http://en.wikipedia.org/w/index.php?title=Recombination_map&amp;action=edit&amp;redlink=1" data-url="http://en.wikipedia.org/w/index.php?title=Recombination_map&amp;action=edit&amp;redlink=1">recombination mapping</a>. It is also the only molecular marker to provide clues about which alleles are more closely related.<a href="http://en.wikipedia.org/wiki/Microsatellite#cite_note-Goldstein-2#cite_note-Goldstein-2" data-url="http://en.wikipedia.org/wiki/Microsatellite#cite_note-Goldstein-2#cite_note-Goldstein-2">[3]</a></p>
		<p id="id41035709" style="text-align: justify;">Microsatellites owe their variability to an increased rate of mutation compared to other neutral regions of DNA. These high rates of mutation can be explained most frequently by&nbsp;<a href="http://en.wikipedia.org/wiki/Slipped_strand_mispairing" data-url="http://en.wikipedia.org/wiki/Slipped_strand_mispairing">slipped strand mispairing</a>&nbsp;(slippage) during&nbsp;<a href="http://en.wikipedia.org/wiki/DNA_replication" data-url="http://en.wikipedia.org/wiki/DNA_replication">DNA replication</a>&nbsp;on a single DNA strand. Mutation may also occur during&nbsp;<a href="http://en.wikipedia.org/wiki/Recombination" data-url="http://en.wikipedia.org/wiki/Recombination">recombination</a>&nbsp;during&nbsp;<a href="http://en.wikipedia.org/wiki/Meiosis" data-url="http://en.wikipedia.org/wiki/Meiosis">meiosis</a>.<a href="http://en.wikipedia.org/wiki/Microsatellite#cite_note-Blouin-3#cite_note-Blouin-3" data-url="http://en.wikipedia.org/wiki/Microsatellite#cite_note-Blouin-3#cite_note-Blouin-3">[4]</a>&nbsp;Some errors in slippage are rectified by&nbsp;<a href="http://en.wikipedia.org/wiki/Proofreading" data-url="http://en.wikipedia.org/wiki/Proofreading">proofreading</a>&nbsp;mechanisms within the&nbsp;<a href="http://en.wikipedia.org/wiki/Cell_nucleus" data-url="http://en.wikipedia.org/wiki/Cell_nucleus">nucleus</a>, but some mutations can escape repair. The size of the repeat unit, the number of repeats and the presence of variant repeats are all factors, as well as the frequency of&nbsp;<a href="http://en.wikipedia.org/wiki/Transcription_(genetics)" data-url="http://en.wikipedia.org/wiki/Transcription_(genetics)">transcription</a>in the area of the DNA repeat. Interruption of microsatellites, perhaps due to mutation, can result in reduced polymorphism. However, this same mechanism can occasionally lead to incorrect&nbsp;<a href="http://en.wikipedia.org/wiki/Amplification" data-url="http://en.wikipedia.org/wiki/Amplification">amplification</a>&nbsp;of microsatellites; if slippage occurs early on during PCR, microsatellites of incorrect lengths can be amplified.</p>
		<p id="id41035866" style="text-align: justify;">Microsatellites can be amplified for identification by the&nbsp;polymerase chain reaction&nbsp;(PCR) process, using the unique sequences of flanking regions as&nbsp;primers. DNA is repeatedly denatured at a high temperature to separate the double strand, then cooled to allow annealing of primers and the extension of nucleotide sequences through the microsatellite. This process results in production of enough DNA to be visible on&nbsp;agarose&nbsp;or&nbsp;polyacrylamide&nbsp;gels; only small amounts of DNA are needed for amplification as thermocycling in this manner creates an exponential increase in the replicated segment<a href="http://en.wikipedia.org/wiki/Microsatellite#cite_note-Griffiths-4#cite_note-Griffiths-4" data-url="http://en.wikipedia.org/wiki/Microsatellite#cite_note-Griffiths-4#cite_note-Griffiths-4">[5]</a>.</p>
		<p id="id41035965" style="text-align: justify;">VNTR&nbsp;The longest DNA repeats are A Variable Number Tandem Repeats (or VNTR). This is a location in a genome where a short nucleotide sequence is organized as a tandem repeat. These can be found on many chromosomes and often show variations in length between individuals. Each variant acts as an inherited allele, allowing them to be used for personal or parental identification. Their analysis is useful in genetics and biology research, forensics, and DNA fingerprinting. VNTR loci are hypervariable loci or minisatellite sequences, which vary in number of repeats of short (16-300 bp) core segment.</p>
		<p style="text-align: justify;">&nbsp;</p>
		<p style="text-align: justify;"><img src="./images/graphics6.png" alt="" width="302" height="272" data-mime-type="image/png" /> </p>
		<p style="text-align: justify;">&nbsp;</p>
		<p id="id41036072" style="text-align: justify;">VNTR have high levels of polymorphism, many alleles and usually can be visualized by Southern blotting or PCR as shown below. VNTR loci are applied in DNA fingerprinting, forensic paternity and linkage analysis.</p>
		<p style="text-align: justify;">&nbsp;</p>
		<p style="text-align: justify;"><img src="./images/graphics8.png" alt="" width="244" height="288" data-mime-type="image/png" /></p>
		<p style="text-align: justify;">&nbsp;</p>
		<p id="id41036167" style="text-align: justify;">One can see a good PowerPoint presentation describing STRs and SSRs and their applications in MITOPENCOURSEWARE</p>        
		</div>
      </div>
    </div>

    <footer class="footer">
      <div class="container">
    <p>Vietnam Open Educational Resources (VOER) is supported by the Vietnam Foundation</p>
    <p>Content license: Creative Commons CC by version 3.0.
    <p>Contact us: <a href="#">voer@voer.edu.vn</a></p>
      </div>
    </footer>

    <script src="js/jquery-latest.js"></script>
    <script src="bootstrap/js/bootstrap.min.js"></script>
    <script src="js/jquery-ui-1.10.1.custom.js"></script>
    <script src="js/main.js"></script>
	<script type="text/javascript" src="js/jquery.js"></script>
	<script type="text/javascript" src="js/jRating.jquery.js"></script>
	<script type="text/javascript">
		$(document).ready(function(){
			$('.basic').jRating();
		});
	</script>
  </body>
</html>
